src/components/views/ComparisonReportList/ComparisonReportList.js

```diff
@@ -368,7 +377,18 @@ const ComparisonReportList = (
           visibleColumns={['title', 'availableVia', 'coverage', 'resourceA', 'resourceB', 'overlap']}
         />
       </div>
-      <div><Button fullWidth onClick={onNeedMoreData}>Load all</Button></div>
+      {totalCount > 200 ?
+        <div>
+          <Button
+            disabled={report.length === totalCount}
+            fullWidth
+            onClick={onNeedMoreData}
+          >
+            Load all
```

Comment:
Apologies for sending this back to you, but have only just noticed that "Load all" here is not translated, same above. Please fix that and then I'm happy to merge through @CalamityC